dlc_developer_config.toml

```diff
@@ -78,14 +78,14 @@ sagemaker_local_tests = false
 # run standard sagemaker remote tests from test/sagemaker_tests
 sagemaker_remote_tests = false
 # run efa sagemaker tests
-sagemaker_efa_tests = false
+sagemaker_efa_tests = true
 # run release_candidate_integration tests
 sagemaker_rc_tests = false
 # run sagemaker benchmark tests
 sagemaker_benchmark_tests = false
 
 # SM remote EFA test instance type
-sagemaker_remote_efa_instance_type = ""
+sagemaker_remote_efa_instance_type = "p5.48xlarge"
```

Comment:
I think you'll need to update this if you want to test this on PT 2.1, as latest should be pointing to PT 2.2